src/main/resources/pipeline/azure/azure.yaml.mustache

```diff
@@ -1,79 +1,68 @@
-# Gradle
-# Build your Java project and run tests with Gradle using a Gradle wrapper script.
-# Add steps that analyze code, save build artifacts, deploy, and more:
-# https://docs.microsoft.com/azure/devops/pipelines/languages/java
-
 name: $(Build.SourceBranchName).$(date:yyyyMMdd)$(rev:.r)
 
 variables:
-  - name: 'artifactory.Repository'
+  - name: 'Artifactory.Repository'
     value: 'build-{{projectName}}'
-  - name: 'projectName'
-    value: '{{projectName}}'
-  - name: 'artifactName'
+  - name: 'Artifact.Name'
     value: '{{projectName}}'
 
 resources:
   - repo: self
+    clean: true
 
 trigger:
   branches:
     include:
       - master
+      - trunk
+      - main
       - feature/*
-      # TODO: Update the main {master/trunk} branch in the list to include it in the branch trigger
 
-jobs:
-  - job: BuildJob
+stages:
+- stage: CI
+  jobs:
+  - job: build
     displayName: 'Build'
     pool:
       name: Build
       demands:
         - java
     steps:
-      - task: CmdLine@2
-        displayName: 'Gradle Wrapper Generation'
-        inputs:
-          script: |
-            gradle wrapper
-      - task: SonarQubePrepare@4
-        displayName: 'Prepare analysis on SonarQube'
-        inputs:
-          SonarQube: SonarQube
-          scannerMode: Other
-          extraProperties: |
-            sonar.projectKey=$(Build.Repository.Name)
-            sonar.projectName=$(Build.Repository.Name)
-            sonar.project.version=$(Build.BuildNumber)
-            sonar.branch.name=$(Build.SourceBranchName)
```

Comment:
What happened with `sonar.branch.name` this is unnecessary?

Reply:
When a build is run on a branch of project, the extension automatically configures the analysis to be pushed to the relevant project branch.